src/Sql/Sql.Test/ScenarioTests/Common.ps1

```diff
@@ -308,11 +308,11 @@ function Get-SqlServerKeyVaultKeyTestEnvironmentParameters ()
 	return @{ rgName = Get-ResourceGroupName;
 			  serverName = Get-ServerName;
 			  databaseName = Get-DatabaseName;
-			  keyId = "https://akvtdekeyvault.vault.azure.net/keys/key1/51c2fab9ff3c4a17aab4cd51b932b106";
-			  serverKeyName = "akvtdekeyvault_key1_51c2fab9ff3c4a17aab4cd51b932b106";
-			  vaultName = "akvtdekeyvault";
+			  keyId = "https://akvtdekeyvaultcl.vault.azure.net/keys/key1/738a177a3b0d45e98d366fdf738840e8";
```

Comment:
Was this vault manually created? Setup should be automated, but at bare minimum please document how you created it

Reply:
Addressed.